eahub/localgroups/views.py

```diff
@@ -75,7 +76,7 @@ def claim_group(request, slug):
         'group_url': "https://{0}/group/{1}".format(get_current_site(request).domain,group.slug),
         'user_email': request.user.email
     })
-    mail_managers(subject, message)
+    send_mail(subject, message, settings.DEFAULT_FROM_EMAIL, recipient_list=settings.LEAN_MANAGERS)
```

Comment:
You have to pass a list of strings in `recipient_list`, not a list of tuples.

I think the easiest solution is to change it to:
`LEAN_MANAGERS = env.list("LEAN_MANAGERS")`
`LEAN_MANAGERS: leanmanager@example.com`

with no associated name. Apologies for not realizing earlier that this would be a problem.

Reply:
Initially I put in just a string and it came back with "recipient_list needs to be either list or tuple". And this is working, I'm receiving the emails on 8001